src/main/webapp/WEB-INF/pages/graph/doi.jsp

```diff
@@ -0,0 +1,76 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: cgong
+  Date: 05/05/2020
+  Time: 13:21
+  To change this template use File | Settings | File Templates.
+--%>
+<%@page pageEncoding="UTF-8" contentType="text/html; charset=UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
+
+<c:import url="../header.jsp"/>
+
+<c:if test="${not empty doiPathways}">
+    <div id="r-responsive-table" class="padding0 top30">
+        <table width="100%" class="reactome" border="0" cellpadding="0" cellspacing="0">
+            <thead>
+            <tr>
+                <th scope="col">Topic</th>
+                <th scope="col">DOI</th>
+                <th scope="col" width="20%">Authors</th>
+                <th scope="col">Released</th>
+                <th scope="col">Revised</th>
+                <th scope="col" width="20%">Reviewers</th>
+                <th scope="col" width="20%">Editors</th>
+            </tr>
+            </thead>
+            <tbody>
+            <c:forEach var="pathway" items="${doiPathways}">
+                <tr>
+                    <td data-label="Topic">
+                        <a href="/content/detail/${pathway.stId}" target="_blank"
+                           title="Show ${pathway.displayName}">${pathway.displayName}</a>[${pathway.species}]
+                    </td>
+                    <td data-label="DOI">${pathway.doi}</td>
+
+                    <td data-label="Authors">
+                        <c:forEach var="person" items="${pathway.authors}" varStatus="status">
+                            <c:set var="fullname" value="${person.surname}, ${person.firstname}"/>
+                            <a href="/content/detail/person/${person.dbId}" target="_blank">${fullname}</a>
+                            <c:if test="${not status.last}">,</c:if>
+                        </c:forEach>
+                    </td>
+
+                    <td data-label="Released">${pathway.releaseDate} ${pathway.releaseStatus}</td>
+
+                    <td data-label="Revised">
+                        <c:if test="${not empty pathway.reviseDate}">
+                            <c:set var="date" value="${pathway.reviseDate}"/>
+                            ${fn:substring(date, 0, 10)}
```

Comment:
hmm nopz, format the date properly with the jstl:fmt

Reply:
oh, good!!!,thank you.